packages/now-cli/src/commands/env/add.ts

```diff
@@ -98,15 +131,59 @@ export default async function add(
 
   if (stdInput) {
     envValue = stdInput;
-  } else if (isSystemEnvVariable(envName)) {
-    envValue = '';
-  } else {
+  } else if (envType === ProjectEnvType.Plaintext) {
     const { inputValue } = await inquirer.prompt({
-      type: 'password',
+      type: 'input',
       name: 'inputValue',
       message: `What’s the value of ${envName}?`,
     });
+
     envValue = inputValue || '';
+  } else if (envType === ProjectEnvType.Secret) {
+    let secretId: string | null = null;
```

Comment:
```suggestion
    let secret: Secret | null = null;
```

Minor nit pick: I would change this to the secret object itself so there is one less variable needed. Then line 177 can assign uid to the envValue.

Reply:
Agreed, I updated it 👍